src/components/tabs/tabs.ts

```diff
@@ -448,8 +448,8 @@ export class Tabs extends Ion implements AfterViewInit {
     // walk backwards through the tab selection history
     // and find the first previous tab that is enabled and shown
     console.debug('run previousTab', this._selectHistory);
-    for (var i = this._selectHistory.length - 2; i >= 0; i--) {
```

Comment:
Can you revert the let/const change?
We are trying to avoid nested let/const since the current transpires sometime generate non-optimal code to simulate the scope by creating a dummy function.